src/main/java/com/programmers/VoucherManagementApplication/voucher/Voucher.java

```diff
@@ -0,0 +1,59 @@
+package com.programmers.VoucherManagementApplication.voucher;
+
+import com.programmers.VoucherManagementApplication.vo.Amount;
+import com.programmers.VoucherManagementApplication.vo.VoucherType;
+
+import java.util.UUID;
+
+public abstract class Voucher {
+
+    private final UUID voucherId;
+    private final VoucherType voucherType;
+    private final Amount amount;
```

Comment:
```suggestion
    protected final UUID voucherId;
    protected final VoucherType voucherType;
    protected final Amount amount;
```

toString과 같이 재정의를 위해서는 protected로 하위 클래스에서 접근 가능하도록 설정하곤 합니다.